### DISCUSSION TITLE
Documentation for v2

### DISCUSSION THREAD

**Discussion by ccatterina** [serverless/serverless#10758] [Q&A] [answered]
Hi, where can I find the documentation for serverless-framework v2? The docs section on serverless website refers to v3, right?

**Comment by pgrzesik** [comment, marked as answer]
Hello @ccatterina - unfortunately at the moment we don't have a good way of browsing v2 docs - I'll bring that up internally, meanwhile you can check out raw `*.md` files on `v2` branch in Framework repository.

**Reply by ccatterina** [reply]
👍  Thank you!
